tempesta_fw/cache.c

```diff
@@ -1294,6 +1291,7 @@ tfw_cache_start(void)
 	TFW_WQ_CHECKSZ(TfwCWork);
 	for_each_online_cpu(i) {
 		TfwWorkTasklet *ct = &per_cpu(cache_wq, i);
+
```

Comment:
There are multiple occurrences where an extra empty line is inserted. However, while this may not be written explicitly in the coding style guidelines, the motivation for NOT having an empty line here is simple. Yes, a variable if defined here. A block of definitions of variable is usually separated with an empty line from the code. However, a variable is also assigned to here at the same time, so it may be thought of as part of the code. When it's one/two/three variables like that, both defined and assigned to at the same time, they are usually treated as part of the code. That makes the code more compact as a whole.

Please trace the changes like that that you had made. It's best if you return it to the way it was before.